frontend/src/metabase/visualizations/components/settings/ChartNestedSettingSeriesMultiple.jsx

```diff
@@ -52,7 +49,11 @@ export default class ChartNestedSettingSeriesMultiple extends React.Component {
                     className="flex-full ml1 align-self-stretch"
                     // set vertical padding to 0 and use align-self-stretch to match siblings
                     style={{ paddingTop: 0, paddingBottom: 0 }}
+                    data-testid="series-name-input"
```

Comment:
Same observation about `aria-label`.

Reply:
I'll add the aria-label here as it's a control